packages/blocks/src/_common/icons/text.ts

```diff
@@ -1468,3 +1468,8 @@ export const EmbedEdgelessIcon = html`<svg
     fill="#77757D"
   />
 </svg> `;
+
+export const CommentIcon = svg`<svg width="20" height="20" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We don't support the comment function yet. The design team should have updated the design. The comment button is currently not needed.

By the way, when we add new icon, we'd better set the fill to 'currentColor', and set the color in icon container, thus the icon color can be followed with the theme.